eZ/Publish/Core/Persistence/Cache/UrlAliasHandler.php

```diff
@@ -62,11 +65,30 @@ public function publishUrlAliasForLocation(
             $updatePathIdentificationString
         );
 
-        $this->cache->invalidateTags([
-            $this->cacheIdentifierGenerator->generateTag(self::URL_ALIAS_LOCATION_IDENTIFIER, [$locationId]),
-            $this->cacheIdentifierGenerator->generateTag(self::URL_ALIAS_LOCATION_PATH_IDENTIFIER, [$locationId]),
-            $this->cacheIdentifierGenerator->generateTag(self::URL_ALIAS_NOT_FOUND_IDENTIFIER),
-        ]);
+        try {
+            $existingLocationAliases = $urlAliasHandler->listURLAliasesForLocation($locationId);
+        } catch (BadStateException $e) {
+            $existingLocationAliases = [];
+        }
+
+        $existingLocationAliasesTags = [];
+        foreach ($existingLocationAliases ?? [] as $existingAlias) {
```

Comment:
Is it possible for `$existingLocationAliases` to be null?

Reply:
No, this was just some leftover from before try/catch.